cli/smartcontract/contract_test.go

```diff
@@ -213,6 +213,110 @@ func ToMap(a []testcontract.MyPair) map[int]string {
 `, string(bs))
 }
 
+func TestDynamicWrapper(t *testing.T) {
+	smartcontract.ModVersion = "v0.0.0"
+
+	tmpDir := t.TempDir()
+	e := testcli.NewExecutor(t, true)
+
+	ctrPath := filepath.Join(tmpDir, "testcontract")
+	e.Run(t, "neo-go", "contract", "init", "--name", ctrPath)
+
+	cfgPath := filepath.Join(ctrPath, "neo-go.yml")
+	bindingsPath := filepath.Join(tmpDir, "bindings.yml")
+	manifestPath := filepath.Join(tmpDir, "manifest.json")
+	cmd := []string{"neo-go", "contract", "compile", "--in", ctrPath}
+
+	goMod := filepath.Join(ctrPath, "go.mod")
+	data, err := os.ReadFile(goMod)
+	require.NoError(t, err)
+
+	i := bytes.IndexByte(data, '\n')
+	data = append([]byte("module myimport.com/testcontract"), data[i:]...)
+
+	wd, err := os.Getwd()
+	require.NoError(t, err)
+	data = append(data, "\nreplace github.com/nspcc-dev/neo-go/pkg/interop => "...)
+	data = append(data, filepath.Join(wd, "../../pkg/interop")...)
+	require.NoError(t, os.WriteFile(goMod, data, os.ModePerm))
+
+	cmd = append(cmd, "--config", cfgPath,
+		"--out", filepath.Join(tmpDir, "out.nef"),
+		"--manifest", manifestPath,
+		"--bindings", bindingsPath)
+
+	e.Run(t, cmd...)
+	e.CheckEOF(t)
+	require.FileExists(t, bindingsPath)
+
+	deployedConctract := testcli.DeployContract(t, e, filepath.Join(ctrPath, "main.go"), filepath.Join(ctrPath, "neo-go.yml"), testcli.ValidatorWallet, testcli.ValidatorAddr, testcli.ValidatorPass)
```

Comment:
Firstly deploy `verify.go` contract and save its hash to variable. See the existing helper:
```

func deployVerifyContract(t *testing.T, e *testcli.Executor) util.Uint160 {
	return testcli.DeployContract(t, e, "../smartcontract/testdata/verify.go", "../smartcontract/testdata/verify.yml", testcli.ValidatorWallet, testcli.ValidatorAddr, testcli.ValidatorPass)
}
```